Containerfile.rust-dev

```diff
@@ -11,7 +11,7 @@ RUN chown dev:dev /home/dev
 # From here we run everything as dev user
 USER       dev
 
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.81.0 -y
```

Comment:
I don't know Rust versioning, but would something like `latest` make sense?

Reply:
I prefer to stick to a version like we do for Go https://github.com/openshift/insights-operator/blob/5e2dd6f1e127c4108a70e94d13debf333a751b7a/go.mod#L5

By using `latest` we would have unreproduceable builds and updates that we might break the code.

Reply:
OK OK :)